RELEASE.md

```diff
@@ -27,41 +29,158 @@ download the latest release candidate CLI from `packages.chef.io`. You
 **must** have run the `/expeditor` Slack command above _before_
 downloading the package!
 
-Alternatively, you can use our "curlbash" installer, specifying the
-`staging` channel as an argument:
+Alternatively, you can either use our "curlbash" installation script
+to install the above packages, or (if you're running on a system that
+already has and older version of `hab` installed) you can upgrade
+using `hab` itself, since the same CLI is available from
+`packages.chef.io` as well as Builder.
+
+
+#### Linux
+
+Run either of the following:
+
+``` sh
+curl https://raw.githubusercontent.com/habitat-sh/habitat/master/components/hab/install.sh \
+    | sudo bash -s -- -c staging
+```
+
+```sh
+sudo hab pkg install core/hab --binlink --force --channel=staging
+```
+
+#### Linux, Kernel 2
+
+Run either of the following commands:
+
+``` sh
+curl https://raw.githubusercontent.com/habitat-sh/habitat/master/components/hab/install.sh \
+    | sudo bash -s -- -c staging -t x86_64-linux-kernel2
+```
+(Note the addition of the target option on the "curlbash" command! Without this you will end up with
+the Kernel 3 version, and things won't work properly!)
```

Comment:
Nit: There is no Kernel 3 for hab :smile:    `x86_64-linux`, `modern linux cli`, `default linux cli` would be more accurate but are a bit of a mouthful.

Reply:
![](https://media.giphy.com/media/yvBAuESRTsETqNFlEl/giphy.gif)

Reply:
Fixed in https://github.com/habitat-sh/habitat/pull/7365/commits/08e67508dea3202bda9db9a8d324ebc89f728559